uwblueprint/starter-code-v2
# Swagger For Python

## Notion ticket link

[Set up Swagger UI for REST APIs](https://www.notion.so/uwblueprintexecs/Task-Board-db95cd7b93f245f78ee85e3a8a6a316d?p=48d82954b93c4fb8a9760a379a0fa45d)



## Implementation description
Basically the exact same as Alex's PR but in Python. The Swagger json file was renamed to snake_case.



## Steps to test
1. Start the server locally with docker-compose up --build (currently it works with mongo + auth, other combos can be tested after scrubbing) * Note that you may need to change entity_routes.py file to use entity_service_mg
2. Navigate to http://localhost:5000/api-docs
3. Test the login endpoint, then add the access_token using the Authorize button in the top right
4. Test out the remaining endpoints, similar to Postman



## What should reviewers focus on?
* documentation_routes.py
* Testing if endpoints work


## Checklist
